examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1716,12 +1716,15 @@ GLTFExporter.prototype = {
 
 			var joints = [];
 			var inverseBindMatrices = new Float32Array( skeleton.bones.length * 16 );
+			var temporaryBoneInverse = new THREE.Matrix4();
```

Comment:
Modules do not use the THREE namespace.

Reply:
> Notice this kind of errors does not happen if you use the mentioned script.

`node utils/modularize.js`, correct? I committed the result of running that script. I'll look into why it generated the incorrect output. Perhaps I have something set up incorrectly.

Reply:
> I committed the result of running that script.

Oh dear, `Matrix4` is on the ignore list here:

https://github.com/mrdoob/three.js/blob/6748b35cd034615ef3a45afb6a68f5b66ce301c1/utils/modularize.js#L38

You just have to remove the entry and run the script again.

Sorry for assumption but sometimes users just copy/paste the code from one file into another.

Reply:
Ok, I've removed `Matrix4` from the ignore list, run the script, and committed the result (including the change to `modularize.js`)

It looks like it also changed an unrelated documentation reference elsewhere in `GLTFExporter`. I'm not sure if this is desirable or not.